src/js/apps/globals/nav/nav_items.js

```diff
@@ -0,0 +1,150 @@
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import { i18n } from 'js/views/globals/app-nav/app-nav_views';
+
+const appNavMenu = new Backbone.Collection([
+  {
+    onSelect() {
+      window.open('https://help.roundingwell.com/');
+    },
+    text: i18n.mainNav.help,
+    icon: {
+      type: 'far',
+      icon: 'life-ring',
+    },
+  },
+  {
+    onSelect() {
+      Radio.request('auth', 'logout');
+    },
+    text: i18n.mainNav.signOut,
+    icon: {
+      type: 'fas',
+      icon: 'right-from-bracket',
+    },
+  },
+]);
+
+const adminNavMenu = new Backbone.Collection([
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'dashboards:all');
+    },
+    id: 'DashboardsApp',
+    text: i18n.adminNav.dashboards,
+    icon: {
+      type: 'far',
+      icon: 'gauge',
+    },
+  },
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'programs:all');
+    },
+    id: 'ProgramsApp',
+    text: i18n.adminNav.programs,
+    icon: {
+      type: 'far',
+      icon: 'screwdriver-wrench',
+    },
+  },
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'clinicians:all');
+    },
+    id: 'CliniciansApp',
+    text: i18n.adminNav.clinicians,
+    icon: {
+      type: 'far',
+      icon: 'users-gear',
+    },
+  },
+]);
+
+const patientsAppWorkflowsNav = new Backbone.Collection([
+  {
+    id: 'owned-by',
```

Comment:
The one new thing I added to each model in this `patientsAppWorkflowsNav` collection here. It represents the `worklistId` each of these belongs to.

This `id` is used by `worklist_app.js` to determine which icon/s to use in the `ListTitleView`.